src/setup-beam.js

```diff
@@ -173,13 +198,25 @@ async function getOTPVersion(otpSpec0, osVersion) {
   return otpVersions.get(otpVersion) // from the reference, for download
 }
 
-async function getElixirVersion(exSpec0, otpVersion0) {
+/**
+ * Return the elixir version from a specification and an OTP version.
+ *
+ * Process the OTP version and the Elixir specification to find the version
+ * in the list of available elixir versions.
+ *
+ * @param exSpec0 Elixir specification. Can be the empty string.
```

Comment:
What does the empty string serve apart from the test that was included to have it there? Can consumers now simply not have a version? I don't understand the "why" (which is what I'd expect I would).

Reply:
The rationale behind this is now explained in another comment. Long story short... a change to the code exposed this, whereas before it was hidden behind another `if`.